third_party/llvm-project/libcxx/src/algorithm.cpp

```diff
@@ -11,9 +11,24 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
+// The definition for std::__bit_log2 is in 
+// llvm-project/libcxx/include/__bit/bit_log2.h and is gaurded by _LIBCPP_STD_VER >= 20
+// Since we are building with C++17 we are adding _LIBCPP_STD_VER >= 20
+
+
+
 template <class Comp, class RandomAccessIterator>
 void __sort(RandomAccessIterator first, RandomAccessIterator last, Comp comp) {
+
+#if defined USE_COBALT_CUSTOMIZATIONS  
```

Comment:
Do we need this, or is the _LIBCPP_STD_VER sufficient ? Note that in the base/net update feature branch `_LIBCPP_STD_VER` issue is separately worked on, we need to be consistent about setting it. There's an open PR somewhere for it

Reply:
I wanted to explicitly say this is a cobalt customization , so someone updating llvm next time knows this while resolving the conflict.